src/core/src/sim/mc/moves/resize.rs

```diff
@@ -54,12 +54,23 @@ impl MCMove for Resize {
 
         // Get the largest cutoff of all intermolecular interactions in the
         // system.
-        // TODO: include electrostatic interactions
-        self.rc_max = system.interactions()
+
+        // Go through global interactions
+        let rc_glob = system.interactions()
+                            .globals()
+                            .iter()
+                            .map(|i| i.cutoff())
+                            .filter_map(|rc| rc)
+                            .fold(f64::NAN, f64::max);
```

Comment:
Yes, that's what I meant. Do we handle the case where there is no cutoff when comparing in the Resize move?